components/Common/ShellBox/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+import styles from './index.module.scss';
+
+interface Props {
+  textToCopy: string;
```

Comment:
I think instead of having a `textToCopy` prop, we should infer the textToCopy based on the children, which in theory should be doable with `ReactDomServer.renderToString` (you can calculate this value as memoised value (`useMemo`) when the children changes

Reply:
I had answers you [here](https://github.com/nodejs/nodejs.org/pull/5234#discussion_r1159950100)